features/dd-sdk-android-session-replay/src/test/kotlin/com/datadog/android/utils/InternalViewUtils.kt

```diff
@@ -0,0 +1,24 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.utils
+
+internal fun isCloseToOrGreaterThan(firstItem: Int, secondItem: Int): Boolean {
+    return secondItem > firstItem || isCloseTo(firstItem, secondItem)
+}
+internal fun isCloseToOrLessThan(firstItem: Int, secondItem: Int): Boolean {
+    return secondItem < firstItem || isCloseTo(firstItem, secondItem)
+}
+
+// add tolerance to mitigate flakiness in tests with Long casting
+internal fun isCloseTo(firstItem: Int, secondItem: Int, tolerance: Double = 0.01): Boolean {
```

Comment:
minor: maybe it would look even better as extension functions? `Int.isCloseToOrGreaterThan`, etc.